sys/shell/commands/sc_gnrc_udp.c

```diff
@@ -105,11 +105,13 @@ static void _send(const char *addr_str, const char *port_str,
          * => use temporary variable for output */
         printf("Success: sent %u byte(s) to [%s]:%u\n", payload_size, addr_str,
                port);
-#if IS_USED(MODULE_ZTIMER_MSEC)
-        ztimer_sleep(ZTIMER_MSEC, delay);
+        if (num) {
+#if IS_USED(MODULE_ZTIMER_USEC)
+            ztimer_sleep(ZTIMER_USEC, delay);
 #else
```

Comment:
How about using USEC if it is there and else divide by 1k
```suggestion
#if IS_USED(MODULE_ZTIMER_USEC)
            ztimer_sleep(ZTIMER_USEC, delay);
#elif IS_USED(MODULE_ZTIMER_MSEC)
            ztimer_sleep(ZTIMER_MSEC, delay/USEC_PER_MSEC+((delay%USEC_PER_MSEC)?1:0));
#else
```